README.rst

```diff
@@ -141,6 +141,29 @@ Here we simply refer to the id attribute on the ami object when creating the ``a
 This works by having a custom ``__getattr__`` function on our ``Data`` and ``Resource`` objects that will turn any attribute access for an attribute name that doesn't exist into the Terraform interpolation syntax.
 
 
+Backend
+-------
+
+Configuring a backend happens in the `Terraform` object. See `Configuring a Terraform Backend`_ for more details.
+
+Bellow we are using an S3 Backend:
+
+.. code-block:: python
+
+    Terraform(
+        dict(
```

Comment:
You shouldn't need to pass in a `dict` here.

For example, ours looks like:

```python
    return Terraform(
        backend=dict(
            ...
```

Reply:
Ah correct! it worked without the first `dict`. 

I updated the PR accordingly.